scripts/update-vendor.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set +eux
+
+# Variables for version updates
+DUMBINIT_VERSION="1.2.2"
+SUPERVISORD_VERSION="0.5"
+##
+VDUMBINIT_VERSION="v${DUMBINIT_VERSION}"
+VSUPERVISORD_VERSION="v${SUPERVISORD_VERSION}"
+REQUIREMENTS_DIR="$(pwd)/vendor"
+
+update_fix_permissions() {
+    echo "Downloading fix permissions script"
+    FIX_PERMISSIONS_SCRIPT="${REQUIREMENTS_DIR}/fix-permissions";
+    curl -o $FIX_PERMISSIONS_SCRIPT https://raw.githubusercontent.com/sclorg/s2i-base-container/master/core/root/usr/bin/fix-permissions;
+    chmod +x $FIX_PERMISSIONS_SCRIPT;
+}
+
+download_tar() {
+    DWN=$1
```

Comment:
can we have more descriptive names for this variables¨?

Reply:
ok